internal/models/source_model.go

```diff
@@ -0,0 +1,20 @@
+package models
+
+type Source struct {
+	// Required auto-generated PK.
+	ID uint64 `db:"id" json:"id"`
+
+	// Source ID in sources
+	SourceID string `db:"source_id" json:"source_id"`
+
+	// User-facing name. Required.
+	Name string `db:"name" json:"name"`
+
+	// Organization ID. Required.
+	OrgID string `db:"org_id" json:"org_id"`
+
+	// Authentication ID in sources lined to a spesific source in order to fetch the ARN
+	AuthID string `db:"auth_id" json:"auth_id"`
```

Comment:
Is this it: https://github.com/RedHatInsights/sources-api-go/blob/main/model/authentication.go#L17

If API allows, I would prefer to use `DbID` which is very much more efficient uint64 (vs string).